bigchaindb/processes.py

```diff
@@ -54,8 +55,16 @@ def start():
     logger.info('WebSocket server started')
     p_websocket_server = mp.Process(name='ws',
                                     target=websocket_server.start,
-                                    args=(events_queue,))
+                                    args=(pubsub.get_subscriber_queue(),))
     p_websocket_server.start()
 
     # start message
     logger.info(BANNER.format(bigchaindb.config['server']['bind']))
+
+    for name, plugin in load_block_publisher_plugins(bigchaindb.config.get('block_publishers', [])):
```

Comment:
Should we use a more generic name instead of `block_publishers`. In the feature we may update this to emit other types of events.